packages/components/src/internal/components/auditlog/models.ts

```diff
@@ -101,7 +102,7 @@ export class TimelineEventModel extends Record({
     }
 
     // timezoneStr used for jest test only, to accommodate teamcity timezone difference
-    static create(raw: any, timezoneStr?: string): TimelineEventModel {
+    static create(raw: any, timezoneStr?: string, inheritedFields?: string[]): TimelineEventModel {
```

Comment:
is `inheritedFields` an array of field names or fieldKeys? if fieldKeys, it might be worth calling the parameter `inheritedFieldKeys` or something along those lines just to make it clear.